History.txt

```diff
@@ -1,3 +1,53 @@
+Documentation and download available at https://www.FreeRTOS.org/
+
+Changes between FreeRTOS-plus-TCP V4.0.0 and V3.1.0 released August 10, 2023
+	+ Unified code for both IPv4 and IPv6 protocols.
+        + Multiple Interface and Multiple Endpoint support
+		1. The stack supports multiple Physical Interfaces.
+		2. The stack also supports multiple endpoints (or Virtual interfaces) on the same physical interface.
+	(Reference: https://www.freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/freertostcp-multiple-interfaces.html).
+        + New WinSim demo to support IPv4, IPv6 and multi-interface together.
+	(Reference: https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo)
+	+ Build separation feature introduced to achieve low memory footprint. Supported build combinations:
+		    1. IPv4 + UDP
+		    2. IPv4 + UDP + TCP
+		    3. IPv6 + UDP
+		    4. IPv6 + UDP + TCP
+		    5. IPv4 + IPv6 + UDP
+		    6. IPv4 + IPv6 + UDP + TCP
+	+ Support added in FreeRTOS-plus-TCP git repository CI tool to validate various build configurations before PR merge.
+	+ Achieved 100% unit test coverage for both IPv4 and IPv6.
+	+ MISRA c-2012 compliance for both IPv4 and IPv6.
+	+ IPv6 code is protocol tested using Maxwell protocol tester.
+	+ Memory safety proofs for IPv6 code using[ CBMC automated reasoning tool ] ( https:/*www.cprover.org/cbmc/).
```

Comment:
https:/*www.cprover.org/cbmc/ - Can the * be removed?

Reply:
txt file does not anyway support links so it think [text](link) - this format will not work

Reply:
Any reason you feel that it is not needed

Reply:
I think you meant "*". Got it. Let me update

Reply:
Done